api/src/stampy_chat/chat.py

```diff
@@ -340,9 +344,38 @@ def run_query(session_id: str, query: str, history: List[Dict], settings: Settin
         prompt=make_prompt(settings, chat_model, callbacks),
         memory=make_memory(settings, history, callbacks)
     )
+
     if followups:
         chain = chain | StampyChain(callbacks=callbacks)
+
     result = chain.invoke({"query": query, 'history': history}, {'callbacks': []})
+
+    #Validate results
+    contexts=[c['text'] for c in get_top_k_blocks(query, 5)]
+    rag_response={
+            "llm_answer": result['text'],
+            "llm_context_list": contexts
+            }
+
+    benchmark = BenchmarkItem(question=query)
+
+    llm_response = LLMResponse(
+            llm_answer=result['text'],
+            llm_context_list=contexts,
+            benchmark_item=benchmark
+        )
+
+    # Score the responses
+    scorer = ValidateScorer([
+        AnswerConsistencyMetric(),
+        AugmentationAccuracyMetric(),
+        RetrievalPrecisionMetric()
+    ])
+    run = scorer.score_responses([llm_response])
+    print(run.overall_scores)
```

Comment:
```
from stampy_chat import logging
logging.info(run.overall_scores)
```

Reply:
you can also send messages to discord with the logger

Reply:
That would be really cool! I will add that in!